templates/filebeat.yml.erb

```diff
@@ -24,6 +24,12 @@ filebeat:
         <%= field.keys[0] %>: <%= field.values[0] %>
   <% end -%>
 <% end -%>
+<% if prospector['close_removed'] -%>
+      close_removed: true
+<% end -%>
+<% if @close_removed_for_all -%>
```

Comment:
Shouldn't this be move below the prospector loop, just above the registry_file entry, otherwise it will be included in the prospector loop